src/Client.php

```diff
@@ -253,7 +253,7 @@ private function configureDefaults(array $config): void
             $defaults['proxy']['no'] = \explode(',', $cleanedNoProxy);
         }
 
-        $this->config = $config + $defaults;
+        $this->config = (array) $config + (array) $defaults;
```

Comment:
it is impossible for either of these to not be an array

Reply:
The same goes for line 304?

Reply:
Today I had a problem for not having an array sum check on line 304 of this “Client” so I did the casting in the two places where I found the problem, if one of the values ​​is null it will give the error "types not supported operands ", by making this change we will no longer have this problem